src/common/services/UserService/UserService.ts

```diff
@@ -24,6 +26,46 @@ const createUser = async (values: CreateUserType): Promise<Response> => {
   });
 };
 
+const readUsers = async (params: ReadUsersQueryType): Promise<ReadUsersResponseType> => {
+  return await fetch(
+    `${import.meta.env.VITE_API_URL}/users?size=${params?.query?.size as number}&page=${params?.query?.page as number}&sort=${
```

Comment:
why as number ?

Reply:
otherwise it returns 'Invalid type "number | undefined" of template literal expression.'